ingest/validation/validate_metadata.py

```diff
@@ -999,14 +1051,18 @@ def validate_collected_ontology_data(metadata, convention):
                 # immediately return as validation should not continue if ontology file unavailable
                 return
         # split on comma in case this property from the convention supports multiple ontologies
+
         ontology_urls = convention["properties"][property_name]["ontology"].split(",")
 
         for ontology_info in metadata.ontology[property_name].keys():
             ontology_id, ontology_label = ontology_info
+
             try:
+                attribute_type = convention["properties"][property_name]["type"]
                 matched_label_for_id = retriever.retrieve_ontology_term_label(
-                    ontology_id, property_name, convention
+                    ontology_id, property_name, convention, attribute_type
```

Comment:
Minor nit - some places we call this variable `type`, and other places `attribute_type`.  Standardizing would help readability.  I personally like `attribute_type` as it is more informative.